### ISSUE TITLE
As a developer, I'd like the jore4 database migrations to be "flattened" into a single one / few

### ISSUE THREAD

**Issue by jschummer** [HSLdevcom/jore4#923]
Before going into production it would be good to combine the jore4 db migrations into a single one or a few ones, whose contents could be grouped by topic.

Having many migrations separately allowed for developers to get the latest changes into their (local) databases without losing their test data sets. But when going to production, it would be good to combine those. (Note that HSL-specific migrations hsould still be kept separately, or rather: The HSL-specific migrations should be combined into a separate single migration. Also note that this has to be done before going to production, because migrations should not be altered after that.

Implementation could be done by exporting the objects introduced by migrations as SQL.

This procedure will substantially ease getting to know the datamodel and database functionality and will enable other stake holder organizations to onboard more quickly.

duplicate of #1014
